src/main/resources/properties/suite/mobile_app/suite.properties

```diff
@@ -1,3 +1,6 @@
-bdd.story-loader.batch-1.resource-location=story/mobile_app
-bdd.story-loader.batch-1.resource-include-patterns=*.story
-bdd.story-loader.batch-1.resource-exclude-patterns=
+batch-1.resource-location=story/mobile_app
```

Comment:
it makes sense to split PR:
- switch to use non-deprecated properties
- Use new Saucelabs application for demo

Reply:
https://github.com/vividus-framework/vividus-sample-tests/pull/48